database/schema.sql

```diff
@@ -167,6 +167,35 @@ CREATE TABLE updating_email_lists_additions (
   PRIMARY KEY (listname, email)
 );
 
+-- ROTATION TABLES
+
+CREATE TABLE rotation_prefrosh (
+  prefrosh_id INTEGER NOT NULL AUTO_INCREMENT,
+  first_name VARCHAR(255) NOT NULL,
+  last_name VARCHAR(255) NOT NULL,
+  preferred_name VARCHAR(255),
+  image LONGBLOB,
+  bucket ENUM('-2', '-1', '000', '0', '0.5', '1', '1.5', '2', '3') NOT NULL DEFAULT '0',
```

Comment:
I like this, but I think the `bucket_id` field in `votes` should be something else instead (possibly just an enum from -2 to 3). Since there are half buckets and the triple zero bucket, under this schema 1.5 would be a valid vote, which isn't how the system works in real life.